bundles/org.eclipse.e4.ui.css.swt.theme/META-INF/MANIFEST.MF

```diff
@@ -1,7 +1,7 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-SymbolicName: org.eclipse.e4.ui.css.swt.theme;singleton:=true
-Bundle-Version: 0.13.100.qualifier
+Bundle-Version: 0.13.101.qualifier
```

Comment:
We bump version by x.y.z+100 to give room for patch releases. Aka in this case it should become 0.13.200.

Reply:
OK, no problem. I changed it.